### PR TITLE
Add AgentService SyncAgent proto to the idl 

### PR DESCRIPTION
**Pull request by Future-Outlier**
# TL;DR
As title.

## Type
 - [ ] Bug Fix
 - [x] Feature
 - [x] Plugin

## Are all requirements met?

 - [x] Code completed
 - [x] Smoke tested
 - [x] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Follow-up issue
https://github.com/flyteorg/flyteidl/pull/413/files#diff-70bc292dedbce9072f9a4bbd20e9ea6c686baf766bcade96b389dac9f4fd553b

### CODE REVIEW

**Comment by Future-Outlier** [comment]
@pingsutw, Can you help approve this test?
Thanks a lot!

File: protos/flyteidl/admin/agent.proto [hunk 1]
```diff
@@ -86,3 +86,19 @@ message DeleteTaskRequest {
...
+// A message used to do a task
+message DoTaskRequest {
+  // The inputs required to start the execution. All required inputs must be
+  // included in this map. If not required and not provided, defaults apply.
+  // +optional
+  core.LiteralMap inputs = 1;
+  // Template of the task that encapsulates all the metadata of the task.
+  core.TaskTemplate template = 2;
+  // Prefix for where task output data will be written. (e.g. s3://my-bucket/randomstring)
+  string output_prefix = 3;
```
Comment (kumare3) [inline protos/flyteidl/admin/agent.proto:99]: it should be outputs!

Comment (Future-Outlier) [reply protos/flyteidl/admin/agent.proto:99]: yes, I think you are right

Comment (Future-Outlier) [reply protos/flyteidl/admin/agent.proto:99]: Thanks a lot!

**Pull request closed without merging**